Libraries/zephyr/MAX/Source/MAX32655/CMakeLists.txt

```diff
@@ -0,0 +1,171 @@
+# Makefile - Analog Devices SDK
```

Comment:
Should the copyright/license text in these CMakeLists.txt files match the copyrights used in the other files?  Or do these files need the particular format you used?

Reply:
@sihyung-maxim, what do you think?

Reply:
I believe this copyright/license text still works because the license identifier references Apache 2.0, and our `LICENSE` file at the root of the MSDK repo applies to every file and subdirectory within the root.

`.txt` files don't need to follow the same format as the source files as it depends on the context of how they're used - they can be used for documentation, data storing, configuration setting, and even as scripts/code. I think it's more of a matter of whether we want to keep consistency with our files, but I recommend using our default copyrights in this case since it's used in a scripting/make context.

Reply:
It was default copyright lines that we and other vendor added in zephyr files.
But it make sense it be consistent on the hal layer.
Updated please see: https://github.com/analogdevicesinc/msdk/pull/990/commits/632d57984a8d1eef6e426cfc1894203bde314881

Reply:
Agree, be consistent with the rest of the MSDK